src/imputation/tmi_imputation.py

```diff
@@ -26,34 +25,6 @@ def filter_by_column_content(
     return df[df[column].isin(column_content)].copy()
 
 
-def impute_pg_by_sic(df: pd.DataFrame, sic_mapper: pd.DataFrame) -> pd.DataFrame:
-    """Impute missing product groups for companies that do no r&d,
-    where instance = 1 and status = "Form sent out"
-    using the SIC number ('rusic').
-
-    Args:
-        df (pd.DataFrame): SPP dataframe from staging
-        sic_mapper (pd.DataFrame): SIC to PG mapper
-
-    Returns:
-        pd.DataFrame: Returns the original dataframe with new PGs
-        overwriting the missing values
-    """
-
-    df = df.copy()
-
-    # Filter for q604 = No or status = "Form sent out"
-    filtered_data = df.loc[(df["status"] == "Form sent out") | (df["604"] == "No")]
-
-    filtered_data = sic_to_pg_mapper(
-        filtered_data, sic_mapper, target_col="201", formtype=[formtype_long]
-    )
-
-    updated_df = apply_to_original(filtered_data, df)
-
-    return updated_df
-
-
```

Comment:
Note to reviewer- this should have been removed in a previous ticket, it's redundant code.